XMonad/Layout/BinarySpacePartition.hs

```diff
@@ -740,9 +748,12 @@ instance LayoutClass BinarySpacePartition Window where
                               , fmap move          (fromMessage m)
                               , fmap splitShift    (fromMessage m)
                               ]
-          resize (ExpandTowards dir) = growNthTowards dir b
-          resize (ShrinkFrom dir) = shrinkNthFrom dir b
-          resize (MoveSplit dir) = autoSizeNth dir b
+          resize (ExpandTowards dir) = growNthTowards dir 0.05 b
+          resize (ShrinkFrom dir) = shrinkNthFrom dir 0.05 b
+          resize (MoveSplit dir) = autoSizeNth dir 0.05 b
```

Comment:
As far as I understand how `PatternSynonyms` work, these cases are redundant as they're covered by the more general constructors